github/resource_github_branch_protection.go

```diff
@@ -183,7 +193,7 @@ func resourceGithubBranchProtectionRead(d *schema.ResourceData, meta interface{}
 
 func resourceGithubBranchProtectionUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*Organization).client
-	r, b, err := parseTwoPartID(d.Id())
+	repoName, branch, err := parseTwoPartID(d.Id())
```

Comment:
tbh parseTwoPartId could return a struct so this could become `id.RepoName` / `id.Branch`

Reply:
Hm, I'm torn on this... I guess it makes a bit more sense to represent it that way, on the other side it adds 3 characters for each variable occurrence.

I don't have a strong opinion, but I think it's outside of scope of the PRs I opened. 😉